front/src/modules/ui/navigation/mobile-navbar/tab-bar/components/TabBar.tsx

```diff
@@ -0,0 +1,75 @@
+import { useNavigate } from 'react-router-dom';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { useRecoilState } from 'recoil';
+
+import { useCommandMenu } from '@/command-menu/hooks/useCommandMenu';
+import {
+  IconCheckbox,
+  IconList,
+  IconSearch,
+  IconSettings,
+} from '@/ui/display/icon';
+import { isNavbarOpenedState } from '@/ui/layout/states/isNavbarOpenedState';
+import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
+
+const StyledContainer = styled.div<{
+  isMobile: boolean;
+}>`
+  bottom: ${({ theme }) => theme.spacing(4)};
+  display: ${({ isMobile }) => (isMobile ? 'flex' : 'none')};
+  gap: ${({ theme }) => theme.spacing(4)};
+  justify-content: center;
+  position: fixed;
+  width: 100%;
+  z-index: 9999999;
```

Comment:
It isn't used well accoss the app but there is a `theme.lastLayerZIndex`

Reply:
I can't use z-index here u are right, but I will need to cut height at your last height.